hack/tools/tools.go

```diff
@@ -1,10 +1,12 @@
+//go:build tools
```

Comment:
You added this or the `gofmt` added this?

PS: If you are using Go v1.17 which will automatically add this tag.